std/range/package.d

```diff
@@ -3231,18 +3225,42 @@ private:
     else
         alias fun = Fun[0];
 
+    import std.traits: ReturnType, functionAttributes, FunctionAttribute;
```

Comment:
The problem is that things like traits and range.primitives can't be moved because they're used for function constraints :/